build/mobilenet-v1/build.py

```diff
@@ -72,6 +89,7 @@ def select_clk_period(platform):
 def select_build_steps(platform):
     if platform in zynq_platforms:
         return [
+            custom_step_update_model,
```

Comment:
If the function was removed, the step can be deleted from this list and the one alveo platforms as well.

Reply:
Resolved in [c3023fb](https://github.com/Xilinx/finn-examples/pull/109/commits/c3023fbd27e2710e9d3c4367d92c75653a04ee30)